README.md

```diff
@@ -1,41 +1,265 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
 
-# Project Name
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-Portfolio
+Modify this file to match your project and remove sections that don't apply.
 
-## Link Portfolio
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-- Link: https://jlberbesi.github.io/Portfolio/
-- Loom: https://www.loom.com/share/aff3750b834b417787d7ed6c7d56e72b
+OPTIONAL SECTIONS:
+- FAQ
 
-## Built With
+After you're finished please remove all the comments and instructions!
+-->
 
```

Comment:
Kindly remove all default text under each section as it was added to guide you in the content to add , removing it will make you work look clean and professional

Reply:
Thank your for your feedback. All linterns are working fine. It includes Eslint. I attach a proof of it.  <img width="1430" alt="Screenshot 2023-03-11 at 3 59 07 PM" src="https://user-images.githubusercontent.com/88105477/224511374-b20a7816-6603-44f5-a662-961a5bff77b7.png">

Reply:
@souad988 Just a kick reminder those lines mentioned above are part of the comments. They don't even appear in the client side.